microsoft-edge/webview2/get-started/winui.md

```diff
@@ -451,6 +451,20 @@ The following classes aren't accessible in WinUI 3:
 
 * `CoreWebView2EnvironmentOptions`
```

Comment:
This 'API limitations' section is no longer accurate and should be removed.

Reply:
removed section